src/Std/Time/Format.lean

```diff
@@ -0,0 +1,287 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Notation
+import Std.Time.Format.Basic
+import Std.Time.Internal.Bounded
+
+namespace Std
+namespace Time
+namespace Formats
+
+set_option linter.all true
+
+/--
+The ISO8601 format, which is always 24 or 27 characters long, used for representing date and time in
+a standardized format. The format follows the pattern `YYYY-MM-DD'T'hh:mm:ss'Z'`.
+-/
+def ISO8601 : Format .any := date-spec% "YYYY-MM-DD'T'hh:mm:ss.sssZ"
```

Comment:
According to the naming convention all of the names here should start with a lowercase letter.